src/commands/CommandEditAlertParserForm.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import { PropTypes } from '@dhis2/prop-types'
+import {
+    Button,
+    ReactFinalForm,
+    FinalForm,
+    NoticeBox,
+    CenteredContent,
+    CircularLoader,
+    SingleSelectField,
+    SingleSelectOption,
+} from '@dhis2/ui'
+import { useDataQuery, useDataEngine } from '@dhis2/app-runtime'
+import { useHistory } from 'react-router'
+
+import { SMS_COMMAND_LIST_PATH } from '../views'
+import i18n from '../locales'
+import { FormRow } from '../forms'
+import { FieldCommandName } from './FieldCommandName'
+import { FieldCommandParser } from './FieldCommandParser'
+import { FieldCommandConfirmMessage } from './FieldCommandConfirmMessage'
+
+const { Form } = ReactFinalForm
+const { FORM_ERROR } = FinalForm
+
+const query = {
+    smsCommand: {
+        resource: 'smsCommands',
+        id: ({ id }) => id,
+        params: {
+            fields: [
+                'name',
+                'parserType',
+                'receivedMessage',
+                'userGroup[name]',
+            ],
+            paging: 'false',
```

Comment:
I think there are two things wrong here:
- `'false'` (string) should be `false` (boolean)
- But the bigger problem is that we are fetching a single item here, so the `paging` property seems irrelevant

Reply:
> 'false' (string) should be false (boolean)

You're right, I usually use a bool here as well. I blindly copied that from another area in the app. I think we're doing that all over the app. We should change that.

> But the bigger problem is that we are fetching a single item here, so the paging property seems irrelevant

Yeah we should be able to just leave it out here.

Reply:
👍  just to confirm, I'm leaving it out and it's working fine

Reply:
I tried that, didn't work, I used `false` first before as well...

Reply:
Removed the paging.

Reply:
Oh because I'm using the boolean in the scheduler as well. I think it works there. Let me check

Reply:
Yeah here for example: https://github.com/dhis2/scheduler-app/pull/23/files#diff-93586d896c98cc73ff851bfabac6e6eaR19. This seems to work